lib/site_settings/label_formatter.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+module SiteSettings
+  class LabelFormatter
+    HUMANIZED_ACRONYMS = %w[
+      acl
+      ai
+      api
+      bg
+      cdn
+      cors
+      cta
+      dm
+      eu
+      faq
+      fg
+      ga
+      gb
+      gtm
+      hd
+      http
+      https
+      iam
+      id
+      imap
+      ip
+      jpg
+      json
+      kb
+      mb
+      oidc
+      pm
+      png
+      pop3
+      s3
+      smtp
+      svg
+      tl
+      tl0
+      tl1
+      tl2
+      tl3
+      tl4
+      tld
+      txt
+      url
+      ux
+    ].freeze
+
+    HUMANIZED_MIXED_CASE = [
+      ["adobe analytics", "Adobe Analytics"],
+      %w[android Android],
+      %w[chinese Chinese],
+      %w[discord Discord],
+      %w[discourse Discourse],
+      ["discourse connect", "Discourse Connect"],
+      ["discourse discover", "Discourse Discover"],
+      ["discourse narrative bot", "Discourse Narrative Bot"],
+      %w[facebook Facebook],
+      %w[github GitHub],
+      %w[google Google],
+      %w[gravatar Gravatar],
+      %w[gravatars Gravatars],
+      %w[ios iOS],
+      %w[japanese Japanese],
+      %w[linkedin LinkedIn],
+      %w[oauth2 OAuth2],
+      %w[opengraph OpenGraph],
+      ["powered by discourse", "Powered by Discourse"],
+      %w[tiktok TikTok],
+      %w[tos ToS],
+      %w[twitter Twitter],
+      %w[vimeo Vimeo],
+      %w[wordpress WordPress],
+      %w[youtube YouTube],
+    ].freeze
+
+    class << self
+      def description(setting)
+        I18n.t("site_settings.#{setting}", base_path: Discourse.base_path, default: "")
+      end
+
+      def humanized_name(setting, setting_label: nil)
```

Comment:
This `setting_label` argument probably isn't needed, not sure why exactly it was there in the JS